src/phase-banner.jsx

```diff
@@ -3,6 +3,14 @@ import PropTypes from 'prop-types';
 
 class PhaseBanner extends React.Component {
 
+  renderContent() {
+    if (this.props.content) {
```

Comment:
Upto you if you change this (it's more a preference which we'll probably come along a lot and should decide what we want) but I would typically use `children` here. Theres no real advantage/disadvantage to either, just preference and consistency.

Then it would be the following for a custom message. 
```jsx
<PhaseBanner phase="beta">This is a new service</PhaseBanner>
// rather than
<PhaseBanner phrase="beta" content="This is a new service" />
```

Reply:
That makes sense. Will change it.